cmd/kube-scheduler/app/options/configfile.go

```diff
@@ -82,7 +82,8 @@ func encodeConfig(cfg *config.KubeSchedulerConfiguration) (*bytes.Buffer, error)
 
 // LogOrWriteConfig logs the completed component config and writes it into the given file name as YAML, if either is enabled
 func LogOrWriteConfig(fileName string, cfg *config.KubeSchedulerConfiguration, completedProfiles []config.KubeSchedulerProfile) error {
-	if !(klog.V(2).Enabled() || len(fileName) > 0) {
+	klogV := klog.V(2)
```

Comment:
I am using `klogV` as variable name instead of `klogV2` because then if the verbosity ever should get changed, only this line needs to be updated.

Reply:
Does it make sense to add it as a comment in the code itself?

Reply:
Each time a `klogV` variable is defined, including `if klogV := klog.V(...); klogV.Enabled()` lines?

I just pointed this code pattern out in a review comment because I wanted to draw attention to it. Normally I would just have used it without further comments.

Reply:
I've added comments for the more complex functions, but not for the simple `f klogV := klog.V(...); klogV.Enabled()`.

@shivanshu1333 please take another look.